Makefile

```diff
@@ -90,21 +94,60 @@ ml-lint:
 	cd $(POETRYPATH) && poetry run vulture
 
 ml-nitta:
-	echo 'Model:' $(ML_MODEL)
+	echo 'Model for Synthesis: ' $(ML_MODEL)
 	$(POETRY) shell
 	MODELS_DIR=$(ML_MODEL_PATH) PYTHONPATH=$(PYTHONPATH) stack exec nitta -- examples/teacup.lua -s ml_$(ML_MODEL) -p=8080
 
+ml-clean:
+	rm -rfv $(ML_CRAWL_DATA_PATH) $(ML_MODEL_PATH)
+
 ############################################################
 ## docker development image
 ############################################################
 
-docker-dev-build:
+
+docker-dev-build-for-linux-win:
+	docker build \
+		--target development \
+		-f ml/synthesis/Dockerfile \
+		--build-arg HOST_UID=$(id -u) \
+		--build-arg HOST_GID=$(id -g) \
```

Comment:
```suggestion
		--build-arg HOST_UID=$(shell id -u) \
		--build-arg HOST_GID=$(shell id -g) \
```

otherwise it's 
```shell
        --build-arg HOST_UID= \
        --build-arg HOST_GID= \
```

Reply:
Missed this one